### PR TITLE
Add browser upload sdk vs storage sdk

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/storage/comparision.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Here's a table summarizing the key differences between the Browser Upload SDK and the Storage SDK:
+
+| **Feature**          | **Browser Upload SDK**                           | **Storage SDK**                                        |
+| -------------------- | ------------------------------------------------ | ------------------------------------------------------ |
+| Environment          | Browser                                          | Node.js                                                |
+| Installation         | npm or Yarn                                      | npm or Yarn                                            |
+| Purpose              | Upload files directly from the browser           | Upload files or directories in a Node.js environment   |
+| Backend Requirement  | Yes, requires a web server for token generation  | No, operates on the server-side                        |
+| SDK Packages         | @spheron/browser-upload                          | @spheron/storage                                       |
```
Comment (rekpero) [inline pages/storage/comparision.mdx:13]: put npm link here

File: pages/storage/comparision.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| Supported Protocols  | IPFS, Filecoin, Arweave                          | IPFS, Filecoin, Arweave                                |
+| Access Control       | Control over who can use the upload token        | No control over token usage                            |
+| Callbacks            | Optional callbacks for tracking uploads          | Optional callbacks for tracking uploads                |
+| Token Expiry         | Token expires after one upload (10-minute limit) | No token required, operates on the server-side         |
```
Comment (rekpero) [inline pages/storage/comparision.mdx:17]: why no token required? expiry can be set by user when generating the token

**Review by rekpero — CHANGES_REQUESTED** [review]
need some changes

File: pages/storage/comparision.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Browser Upload SDK vs. Storage SDK
```
Comment (rekpero) [inline pages/storage/comparision.mdx:1]: name the sidebar title - `SDK Comparision`

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/comparision.mdx (hunk 3 above)
Comment (aayushmahapatra) [reply pages/storage/comparision.mdx:1]: It's named - `Browser vs Storage` in #102

**Review by rekpero — APPROVED** [review]

**Pull request merged**